setup.py

```diff
@@ -24,6 +24,7 @@
 ]
 
 EXTRAS_REQUIRE = {
+    'ci': ['teamcity-messages'],
```

Comment:
I would prefer to avoid this.

A similar example is `pytest-cov` or `coveralls`. These are not required for the end-user to run tests on the installed binary/source Python package; we only use them on CI. So the CI scripts install them, and they don't appear in setup.py.

Alternate locations would be in requirements.txt, or in run_tests.sh